crypto-ffi/bindings/js/CoreCrypto.ts

```diff
@@ -1691,7 +1691,7 @@ export class CoreCrypto {
      * Generates an E2EI enrollment instance for a "regular" client (with a Basic credential) willing to migrate to E2EI.
      * Once the enrollment is finished, use the instance in {@link CoreCrypto.e2eiRotateAll} to do the rotation.
      *
-     * @param clientId - client identifier with user b64Url encoded & clientId hex encoded e.g. `t6wRpI8BRSeviBwwiFp5MQ:6add501bacd1d90e@example.com`
+     * @param clientId - client identifier with user b64Url encoded & clientId hex encoded e.g. `b7ac11a4-8f01-4527-af88-1c30885a7931:6add501bacd1d90e@example.com`
```

Comment:
Comments are wrong - the client identifier isn't a b64Url anymore from the example but hyphenated UUID string repr